ai_agents/agents/ten_packages/extension/fish_audio_tts_python/extension.py

```
@@ -109,33 +111,32 @@ async def on_deinit(self, ten_env: AsyncTenEnv) -> None:
-        await super().on_data(ten_env, data)
+                    duration_ms = self._calculate_audio_duration_ms()
+                    await self.send_tts_audio_end(
+                        request_id=self.current_request_id,
+                        request_event_interval_ms=request_event_interval,
+                        request_total_audio_duration_ms=duration_ms,
+                        reason=TTSAudioEndReason.INTERRUPTED,
+                    )
+        else:
+            if self.ten_env:
+                self.ten_env.log_warn(
+                    f"No current request found, skipping TTS cancellation."
```

Ruff (0.14.0)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)